### DISCUSSION TITLE
本地编译报错求解

### DISCUSSION THREAD

**Discussion by ghost** [Boos4721/openwrt#127] [Bugs/问题反馈]
ax6，只更改了luci-app，其余没动

make[1]: *** [package/Makefile:110: /home/axbng/openwrt/staging_dir/target-aarch64_cortex-a53_musl/stamp/.package_compile] Error 2
make[1]: Leaving directory '/home/axbng/openwrt'
make: *** [/home/axbng/openwrt/include/toplevel.mk:230：world] 错误 2

**Comment by Boos4721** [comment]
DAMA, any error mesg?

**Reply by ghost** [reply]
我晚些重新编译，若再报错我再反馈

**Reply by ghost** [reply]
> DAMA, any error mesg?

本地编译依旧报错，不明所以，我有空再研究下吧......

**Reply by ghost** [reply]
已编译成功，盲猜nss-ecm的问题
